webpack/src/Components/EmptyState/ExtendedEmptyState.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Button, Alert } from '@patternfly/react-core';
+import EmptyState from 'foremanReact/components/common/EmptyState/EmptyStatePattern';
+import SkeletonLoader from 'foremanReact/components/common/SkeletonLoader';
+import PropTypes from 'prop-types';
+
+import { STATUS } from 'foremanReact/constants'
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks'
+import { translate as __ } from 'foremanReact/common/I18n';
+import { FOREMAN_STORYBOOK } from './Constants';
+
+const ExtendedEmptyState = ({ header }) => {
+  // AJAX request using useAPI hook
+  const { response: { description }, status } = useAPI('get', '/plugin_template_description')
```

Comment:
can we possibly use some already existing controller instead of the react global?
Something like `ExampleController` would do the job IMHO.